icons/vtex.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>VTEX</title><path d="M 19.300781 5.585938 L 8.304688 5.585938 C 7.453125 5.585938 6.90625 6.507812 7.308594 7.277344 L 8.40625 9.402344 L 6.414062 9.402344 C 6.15625 9.402344 5.917969 9.539062 5.785156 9.765625 C 5.652344 9.988281 5.644531 10.269531 5.765625 10.5 L 9.300781 17.332031 C 9.429688 17.578125 9.675781 17.730469 9.949219 17.730469 C 10.222656 17.730469 10.46875 17.578125 10.597656 17.332031 L 11.558594 15.488281 L 12.761719 17.816406 C 13.1875 18.632812 14.335938 18.636719 14.761719 17.816406 L 20.269531 7.234375 C 20.660156 6.488281 20.128906 5.585938 19.300781 5.585938 Z M 14.359375 10.105469 L 11.984375 14.671875 C 11.902344 14.832031 11.734375 14.933594 11.554688 14.933594 C 11.375 14.933594 11.207031 14.832031 11.125 14.671875 L 8.769531 10.125 C 8.691406 9.972656 8.695312 9.785156 8.78125 9.636719 C 8.871094 9.484375 9.027344 9.394531 9.199219 9.394531 L 13.941406 9.394531 C 14.109375 9.394531 14.261719 9.484375 14.347656 9.628906 C 14.433594 9.773438 14.4375 9.957031 14.359375 10.105469 Z M 14.359375 10.105469 "/></svg>
```

Comment:
Looks like linting is failing because the icon is not optimized. Try running `npm run svgo -- icons/vtex.svg`.

Reply:
Have added a new pull request here : https://github.com/simple-icons/simple-icons/pull/6989